doc/resources.md

```diff
@@ -0,0 +1,43 @@
+# Resources
+
+These are all the official Nix resources.
+
+Currently there is no established process for how to make new official resources or make existing resources official.
```

Comment:
Because we had no official process, and it was blocking, I started making a couple of judgment calls. We could document that as a fact. Ideally, we would come up with a policy that could be used to make those decisions more predictable than my gut feelings.